src/components/RichTextEditor/helpers/forceToFocusElement.ts

```diff
@@ -0,0 +1,18 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export const forceToFocusNextElement = (event: React.KeyboardEvent, forwards: boolean) => {
+    if (!document.activeElement) {
+        return;
+    }
+    event.preventDefault();
+    const focusableElementSelectors =
+        'button, [href], input, select, textarea, [role="textbox"], [tabindex]:not([tabindex="-1"]';
```

Comment:
```suggestion
        ''a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, embed, [tabindex="0"], [contenteditable]';
```

What do you think to adjust them a bit more, so disabled stuff is not focusable

Reply:
On the focusTrap hook I'm working on, the list of tabbable elements is like this:
export const TABBABLE_ELEMENTS = [
    'input:not([disabled]):not([type=hidden])',
    'select:not([disabled])',
    'textarea:not([disabled])',
    'button:not([disabled])',
    'a[href]',
    'area[href]',
    'summary',
    'iframe',
    'object',
    'embed',
    'audio[controls]',
    'video[controls]',
    '[contenteditable]',
    '[tabindex]:not([tabindex="-1"]):not([disabled])',
].join(':not([hidden]):not([tabindex="-1"]),');

Reply:
Thanks, great list. Implemented it with some tweaks.

Reply:
@imoutaharik thanks for that improved list.